src/com/can/CANLayer.h

```diff
@@ -8,62 +8,71 @@
  * SPDX-License-Identifier: EPL-2.0
  *
  * Contributors:
- *  Malte Grave - initial API and implementation and/or initial documentation
+ *  Malte Grave, Alexander Trojnin - initial API and implementation and/or initial documentation
  *******************************************************************************/
 
 #pragma once
 
 #include <linux/can.h>
 #include <linux/can/raw.h>
 #include <net/if.h>
+
 #include <sockhand.h>
 
 #include "comlayer.h"
 #include "datatype.h"
-#include "fdselecthand.h"
 #include "gensockhand.h"
 #include "parameterParser.h"
+#include "string"
 
-namespace forte::com_infra {
+#include "CANHandler.h"
 
-class CCANComLayer : public CComLayer {
-public:
-  typedef FORTE_SOCKET_TYPE TCANSocketHandle;
+namespace forte
```

Comment:
Since we are using modern C++ version, we can use the C++17 namespace style it's shorter.

https://en.cppreference.com/w/cpp/language/namespace.html

Reply:
Update